app/src/main/res/layout/activity_multimeter.xml

```diff
@@ -369,97 +40,418 @@
+                app:layout_constraintStart_toStartOf="@+id/lower_left_box"
+                app:layout_constraintTop_toBottomOf="@+id/knob_marker">
+
+                <android.support.v7.widget.SwitchCompat
+                    android:id="@+id/selector"
+                    android:layout_width="@dimen/multimeter_length_0"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center"
+                    android:layout_weight="1"
+                    android:checked="false"
+                    android:gravity="center"
+                    android:text="@string/frequency_unit"
+                    android:textStyle="bold"
+                    android:thumbTextPadding="@dimen/multimeter_margin_2" />
+
+                <TextView
+                    android:layout_width="@dimen/multimeter_length_0"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1"
+                    android:gravity="center_horizontal"
+                    android:paddingBottom="@dimen/multimeter_button_text_padding"
+                    android:paddingTop="@dimen/multimeter_button_text_padding"
+                    android:text="@string/count_pulse"
+                    android:textColor="@color/black"
+                    android:textSize="16sp"
+                    android:textStyle="bold" />
+
+            </LinearLayout>
+
+            <View
+                android:id="@+id/knob_marker"
+                android:layout_width="@dimen/multimeter_knob_width"
+                android:layout_height="@dimen/multimeter_knob_height"
+                android:layout_marginBottom="@dimen/multimeter_margin_3"
+                android:layout_marginEnd="@dimen/multimeter_constraint_1"
+                android:layout_marginLeft="@dimen/multimeter_constraint_1"
+                android:layout_marginRight="@dimen/multimeter_constraint_1"
+                android:layout_marginStart="@dimen/multimeter_constraint_1"
+                android:layout_marginTop="@dimen/multimeter_constraint_1"
+                android:background="@drawable/knob_lines"
+                android:visibility="visible"
+                app:layout_constraintBottom_toBottomOf="@+id/freq_count"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center"
+                android:gravity="center"
+                android:text="@string/ch1"
+                android:textColor="@color/colorPrimary"
+                android:textSize="@dimen/control_textsize_tiny"
+                android:textStyle="bold"
+                app:layout_constraintCircle="@id/knobs"
+                app:layout_constraintCircleAngle="345"
```

Comment:
For angles of the text view, you can make an array in resources file. So, it will be easy to change them whenever necessary.

Reply:
@harsh-2711 There is no method to access an array element directly in the layout file ,please follow this link  thus I think there is no use of making the array, I have done the other changes
--

Reply:
@harsh-2711 There is no way to access an array element directly in the layout file, refer this [link](https://stackoverflow.com/questions/30465202/how-do-i-call-string-array-value-in-xml-layout), thus I think there is no use of making the array, I have done the other changes :+1:

Reply:
Yes, @abhinavraj23 . We can only use the complete array at a time and not single entries in XML. So, can you use single string for each angle in resources file as hardcoded dimensions aren't according to best practices.

Reply:
Will do the changes :+1: